coursedata/adventures/de.yaml

```diff
@@ -603,164 +609,195 @@ adventures:
+                    Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
 
                 start_code: |
-                    print 'Drawing figures'
-                    angle is 90
-                    turn angle
+                    print 'Zeichne Figuren'
+                    Winkel is 90
+                    turn Winkel
                     forward 25
 
             5:
                 story_text: |
-                    ## Let's draw
-                    In level 5 you can make a choice with `if`. For example between different types of figures.
+                    ## Lass uns zeichnen
+                    In Level 5 kannst Du mit dem Befehl `if` Entscheidungen treffen.
+                    Diú kannst Dich zum Beispiel zwischen verschiedenen Formen entscheiden.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120
-                    else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
+                    if Form is Dreieck Winkel is 120
+                    else Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print 'Drawing Figures'
-                    figure is ask 'Do you want a square or a triangle?'
-                    if figure is triangle angle is 120 else angle is 90
-                    turn angle
+                    print 'Zeichne Formen'
+                    Form is ask 'Möchtest Du ein Quadrat oder ein Dreieck zeichnen?'
+                    if Form is Dreieck Winkel is 120 else Winkel is 90
+                    turn Winkel
                     forward 25
             8:
                 story_text: |
-                    ## Let's draw
-                    Now that we can repeat several lines, we can make figures more easily.
-                    We only have to set the angle once and then use that variable in the `repeat`.
+                    ## Lass uns zeichnen
+                    Jetzt, da wir einzelne Linien wiederholen können, fällt uns das Zeichnen einzelner Formen leichter.
+                    Wir setzen den Winkel einmalig und nutzen dann die Variable mit `repeat`.
```

Comment:
```suggestion
                    Wir setzen den Winkel einmalig und nutzen dann die Variable im `repeat`.
```

Reply:
With "im `repeat`" I tried to mirror the (similarly imprecise) "in the `repeat`" of the English version, understanding it as a way to say "in the `repeat` block`" without having to spell out the (indeed somewhat technical) "block". Neither are ideal, but for now, I'd go with "im".